### ISSUE TITLE
bug to log to Cryptpad

### ISSUE THREAD

**Issue by guigui46** [xwiki-labs/cryptpad#654]
Since 30 minutes, around 1:00 pm, Paris Time, the CRYPTPAD website doesn't work - impossible to log an account. There's no information on their website, and no one from the company, XWiki SAS, answer.

**Comment by ansuz** [comment]
It seems there was an outage at our datacenter that affected many machines. We're investigating and will have our server back online as soon as possible.

**Comment by guigui46** [comment]
Ok, I understand, many thanks for your answer.

**Comment by radazamarito** [comment]
Hi,

I'm having the same issue with logging it.

**Comment by Lexe0909** [comment]
Hi,  me too, i can't have access to my pad.

**Comment by cyberflaneuse** [comment]
Hello, I am having the same issue. Please let us know when you think it will be resoled. Thank you.

**Comment by ansuz** [comment]
We have access to the affected machine and we're currently performing an integrity check on the disk. Once that finishes we should be able to restore service fairly quickly, but it's hard to tell how much longer the check will take.

**Comment by ansuz** [comment]
I've updated our site to point to our [mastodon](https://social.weho.st/@cryptpad) and [twitter](https://twitter.com/cryptpad) accounts, where we've been posting updates as we've learned more. That will be the best way to stay up to date.

Some time earlier today a malfunction in a water cooling system at our data-center caused the physical server which hosts a number our company's VMs to overheat and go offline. We don't know how many other servers were affected, but ours was not the only one.

We spent some time on the phone with our server host and we were able to eventually get most of our resources connected to a different physical machine.  **All data stored up to the moment the server became unavailable is safe**. Unfortunately, we still have to load that data onto a new machine and reconfigure the new system to work as before.

We're migrating close to a terabyte of data over a network and expect this process to take quite a few hours to complete. We're allowing this process to run overnight and plan to check everything early in the morning (Central European Time). If all goes well we hope to resume normal service by noon (CET).

Thanks for your patience in the meantime.
